<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectModeDlg</class>
 <widget class="QDialog" name="SelectModeDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>368</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>670</width>
    <height>368</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>670</width>
    <height>368</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QRadioButton" name="radioOnlineWallet">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>93</y>
     <width>520</width>
     <height>37</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Online Wallet (recommended for most users) - MWC wallet with a full functionality.
Internet connection is required.</string>
   </property>
  </widget>
  <widget class="control::MwcPushButtonNormal" name="submitButton">
   <property name="geometry">
    <rect>
     <x>354</x>
     <y>303</y>
     <width>157</width>
     <height>40</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioOnlineNode">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>153</y>
     <width>526</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Online Node - MWC node that provides blockchain data for the Cold Wallet</string>
   </property>
  </widget>
  <widget class="control::MwcLabelLarge" name="label_3">
   <property name="geometry">
    <rect>
     <x>204</x>
     <y>12</y>
     <width>259</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Running Mode</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="control::MwcPushButtonNormal" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>159</x>
     <y>303</y>
     <width>157</width>
     <height>40</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioColdWallet">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>195</y>
     <width>523</width>
     <height>34</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Cold Wallet - Offline MWC wallet with limited functionality
(mainly used by exchanges)</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>control::MwcPushButtonNormal</class>
   <extends>QPushButton</extends>
   <header>control/MwcPushButton.h</header>
  </customwidget>
  <customwidget>
   <class>control::MwcLabelLarge</class>
   <extends>QLabel</extends>
   <header>control/MwcLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
